src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -201,6 +221,7 @@ HttpStatusCode mongoSubscribeContext
     LM_T(LmtSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
 
     cacheSemTake(__FUNCTION__, "Inserting subscription in cache");
+    LM_W(("Calling subCacheItemInsert with status == %s", status.c_str()));
```

Comment:
Yes, I use LM_W(("KZ; ...")) for my traces, and then I grep on KZ to remove them. Here it seems I forgot the 'KZ' part so I didn't find it in my grep for removal